### PR TITLE
source-simplesat contribution from aazam-gh

### LINKED ISSUE #46055: [connector-request] Simplesat
This is a **suggestion** to create a connector for **Simplesat** using the UI Connector Builder.
The website for the connector is https://www.simplesat.io.

If you want to try, claim this issue and start working on it following the steps below.

### Steps:
1. Comment in the issue and wait to be assigned to start working on it.
2. Map the API endpoints (get approval to move to 3 step)
3. Describe steps to get credentials.
4. Create the connector using the UI Builder

> [!IMPORTANT]
> Airbyte doesn't have sandbox credentials for this connector. You must have access/credentials to the service provider to create the connector.

> [!CAUTION]
> This is a **suggestion** and there may be cases where creating the connector using the Builder won't be possible. To minimize the risk of investing a lot of time directly in creating the connector, we strongly recommend to follow steps above.

### PR DESCRIPTION
**Pull request by aazam-gh**
## What

Closed https://github.com/airbytehq/airbyte/issues/46055

This PR adds a new connector for source Simplesat (source-simplesat).

Contributor-provided connector description:

This is the setup guide for the Simplesat source connector.
Fixes #46055 

## Details

<details><summary>Expand for list of spec properties and streams</summary>

### Configuration

| Input | Type | Description | Default Value |
|-------|------|-------------|---------------|
| `api_key` | `string` | API Key.  |  |

### Streams
| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
|-------------|-------------|------------|---------------------|----------------------|
| surveys | id | DefaultPaginator | ✅ |  ❌  |
| questions | id | DefaultPaginator | ✅ |  ❌  |

</details>

## Reviewer checklist
- [ ] Run `/format-fix` to resolve any formatting errors
- [ ] Help the contributor upload an icon that follows our icon design guidelines
- [ ] Click into the CI workflows that wait for a maintainer to run them, which should trigger CI runs

### CODE REVIEW

**Comment by marcosmarxm** [comment]
@aazam-gh Can we give a try using the POST `/search` endpoints? E.g: https://api.simplesat.io/api/v1/answers/search

File: airbyte-integrations/connectors/source-simplesat/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,265 @@
+version: 5.7.5
+
+type: DeclarativeSource
+
+description: |
+  This is the setup guide for the Simplesat source connector.
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-simplesat/manifest.yaml:6]: The description is about the provider. Please provide a short paragraph explaining what this source does.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Also need a svg icon of the source.

**Comment by aazam-gh** [comment]
@marcosmarxm added search endpoint.

File: airbyte-integrations/connectors/source-simplesat/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+            page_size: 100
+            start_from_page: 1
+            inject_on_first_request: true
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/questions"
+    search:
+      type: DeclarativeStream
+      name: search
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-simplesat/manifest.yaml:90]: A better name to this stream is `answers`.

File: airbyte-integrations/connectors/source-simplesat/manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,399 @@
+version: 5.8.1
+
+type: DeclarativeSource
+
+description: |
+  Website : https://www.simplesat.io/
+  API Docs : https://documenter.getpostman.com/view/457268/SVfRt7WJ?version=latest
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-simplesat/manifest.yaml:7]: Can you add a textual description instead of links?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-simplesat/manifest.yaml (hunk 3 above)
Comment (aazam-gh) [reply airbyte-integrations/connectors/source-simplesat/manifest.yaml:7]: Sure!

**Comment by aazam-gh** [comment]
@marcosmarxm made the requested changes. :)

File: airbyte-integrations/connectors/source-simplesat/manifest.yaml [hunk 4]
```diff
@@ -0,0 +1,407 @@
...
+  The Simplesat source connector which ingests data from the Simplesat API.
+  Simplesat is a engaging survey tool for service businesses to gather insight and feedback from their customers.
+
+  The source supports 3 of the endpoints from the API (questions, surveys and answers)
+  For more information, checkout the website https://www.simplesat.io/
+  
+  An API key is required for authentication and using this connector.
+  In order to obtain an API key, you must first create a Simplesat account.
+  Once logged-in, you will find your API key in the account settings.
+  You can find more about their API here https://documenter.getpostman.com/view/457268/SVfRt7WJ?version=latest
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-simplesat/manifest.yaml:15]: The documentation is not right, it should be v1 https://documenter.getpostman.com/view/457268/UVRDGRZ2#fc3683b6-2933-425b-b4d1-faa8377de1ac

File: airbyte-integrations/connectors/source-simplesat/manifest.yaml [hunk 5]
```diff
@@ -0,0 +1,407 @@
...
+          pagination_strategy:
+            type: PageIncrement
+            page_size: 100
+            start_from_page: 1
+            inject_on_first_request: true
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/questions"
+    search:
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-simplesat/manifest.yaml:96]: This endpoint must be called answers not search.

Comment (marcosmarxm) [reply airbyte-integrations/connectors/source-simplesat/manifest.yaml:96]: Also you can implement `responses` stream using the same logic.

File: docs/integrations/sources/simplesat.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `api_key` | `string` | API Key.  |  |
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| surveys | id | DefaultPaginator | ✅ |  ❌  |
+| questions | id | DefaultPaginator | ✅ |  ❌  |
```
Comment (marcosmarxm) [inline docs/integrations/sources/simplesat.md:15]: Later update the entry here.

File: docs/integrations/sources/simplesat.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
+# Simplesat
+This is the setup guide for the Simplesat source connector.
```
Comment (marcosmarxm) [inline docs/integrations/sources/simplesat.md:2]: Update the description.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Left some comments

File: docs/integrations/sources/simplesat.md (hunk 7 above)
Comment (aazam-gh) [reply docs/integrations/sources/simplesat.md]: Updated

File: docs/integrations/sources/simplesat.md (hunk 6 above)
Comment (aazam-gh) [reply docs/integrations/sources/simplesat.md:39]: Done

File: airbyte-integrations/connectors/source-simplesat/manifest.yaml (hunk 5 above)
Comment (aazam-gh) [reply airbyte-integrations/connectors/source-simplesat/manifest.yaml]: Implemented

**Comment by aazam-gh** [comment]
@marcosmarxm reviews have been addressed

**Review by marcosmarxm — APPROVED** [review]
Awesome contribution @aazam-gh

**Pull request merged**